README.md

```diff
@@ -150,16 +146,30 @@ TODO([#14](https://github.com/jackbsteinberg/std-toast/issues/14)): create an ex
 
 The element is provided as a [built-in module](https://github.com/tc39/proposal-javascript-standard-library/blob/master/README.md),
 named `"std:elements/toast"`.
+Read more about "pay-for-what-you-use" HTML elements [here](https://github.com/whatwg/html/issues/4697).
 
 ### The `<std-toast>` element
 
 #### Attributes
 
 - [Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)
-- `open`: a boolean attribute, determining whether the toast is visible or not (according to the default styles). By default toasts are not shown.
+- `open`: a boolean attribute, determining whether the toast is visible or not (according to the default styles).
+By default toasts are not shown.
 - `theme`: one of `"default"`, ???, or ???, conveying the semantic priority of the toast, and influencing its styling (both default and user-provided)
     - TODO([#18](https://github.com/jackbsteinberg/std-toast/issues/18)): decide on list of themes to natively support and create and style them.
-- `position`: one of `"top-left"`, `"top-center"`, `"top-right"`, `"middle-left"`, `"middle-center"`, `"middle-right"`, `"bottom-left"`, `"bottom-center"`, or `"bottom-right"`.
+- `position`: default position will be ???
+    - Options for position:
+        - `"top-left"`
+        - `"top-center"`
+        - `"top-right"`
+        - `"middle-left"`
+        - `"middle-center"`
+        - `"middle-right"`
+        - `"bottom-left"`
+        - `"bottom-center"`
+        - `"bottom-right"`
+    - Additionally values `"top-stretch"` and `"bottom-stretch"` will be provided to allow for toasts that span the screen width,
+    and will be treated as the default for `"top-"` and `"bottom-"` toasts on mobile.
```

Comment:
I'm not sure we should be definitive about this particular API choice yet. Maybe instead some kind of TODO about stretch styles and about mobile vs. desktop?

Reply:
Created issue #39 and referenced in commit

Reply:
I removed the `middle-left` and `middle-right` positions, and changed the center position to just be `center` by the way, they seemed like values we didn't need to include